src/python/bcc/table.py

```diff
@@ -559,6 +561,58 @@ def __delitem__(self, key):
             lib.bpf_close_perf_event_fd(self._open_key_fds[key])
         del self._open_key_fds[key]
 
+    def _get_event_class(self):
+        ct_mapping = { 'char': ct.c_char,
+                       'unsigned char': ct.c_ubyte,
+                       'u8': ct.c_ubyte,
+                       'u8 *': ct.c_char_p,
+                       'char *': ct.c_char_p,
+                       'short': ct.c_short,
+                       'unsigned short': ct.c_ushort,
+                       'u16': ct.c_ushort,
+                       'int': ct.c_int,
+                       'unsigned int': ct.c_uint,
+                       'u32': ct.c_uint,
+                       'long': ct.c_long,
+                       'unsigned long': ct.c_ulong,
+                       'long long': ct.c_longlong,
+                       'unsigned long long': ct.c_ulonglong,
+                       'u64': ct.c_ulonglong,
+                       'void *': ct.c_void_p }
```

Comment:
Maybe add s8, s16, s32, s64 as well?
How do we handle `__int128` and `unsigned __int128`? I guess in such cases, the old way is needed where `__int128` is coded as `s64[2]`?

Reply:
s8/16/32/64 added. All other types not listed in this map will cause script
to stop with message "Type: '%s' not recognized. Please use ctypes manually."
As _int128 is rare (I assume), how about coming to this issue later sometime?

Reply:
yes, the old way works great for __int128: